app.js

```diff
@@ -158,4 +158,19 @@ const x = 0;
 if (x === 1) {
   popupWC();
   popupW();
-}
\ No newline at end of file
+}
+
+// form validation
+const formSubmit = document.querySelector('#form');
+formSubmit.addEventListener('submit', (event) => {
+  const emailInput = document.querySelector('#email');
+  const errorMessage = document.querySelector('#error-message');
+
+  if (emailInput.value.toLowerCase() !== emailInput.value) {
+    errorMessage.innerText = 'Please enter lowercase email ID';
+    errorMessage.style.display = 'block';
+    event.preventDefault();
```

Comment:
- [ OPTIONAL]
- Your validation works fine but I would suggest you improve the error message style, the text is to big and the back color does not really represent the error, try with `dack-red`
- Your header scrolls but not some items with it, please either make your header fixed and add a white bg-color so it will not get confused with the page contents or make your  items scroll together with the header

Reply:
Thanks